components/homme/cmake/machineFiles/weaver-bfb.cmake

```diff
@@ -0,0 +1,111 @@
+# Bit-For-Bit Testing CMake Cache File for Weaver (IBM POWER 9 & NVIDIA TESLA V100s)
+# Skyler Ruiter -- 5/30/24
+#
+#  Module list that worked for this cache file (throw into a script load-weaver)
+#
+#  1) cmake/3.25.1    4) py-netcdf4/1.5.8   7) openmpi/4.1.4         10) parallel-netcdf/1.12.3  13) curl/7.87.0
+#  2) git/2.39.1      5) gcc/11.3.0         8) netcdf-c/4.9.0        11) netlib-lapack/3.10.1
+#  3) python/3.10.8   6) cuda/11.8.0        9) netcdf-fortran/4.6.0  12) openblas/0.3.23
+
+
+############################################
+#                                          #
+#                ENV PATHS                 #
+#                                          #
+############################################
+
+# PIO settings
+EXECUTE_PROCESS(COMMAND nf-config --prefix
+  RESULT_VARIABLE NFCONFIG_RESULT
+  OUTPUT_VARIABLE NFCONFIG_OUTPUT
+  ERROR_VARIABLE  NFCONFIG_ERROR
+  OUTPUT_STRIP_TRAILING_WHITESPACE
+)
+#SET (NetCDF_Fortran_PATH "${NFCONFIG_OUTPUT}" CACHE STRING "")
+
+# Set the path manually otherwise /home/projects/e3sm/scream/libs/netcdf-fortran/install/weaver will get grabbed
+SET (NetCDF_Fortran_PATH /projects/ppc64le-pwr9-rhel8/tpls/netcdf-fortran/4.6.0/gcc/11.3.0/openmpi/4.1.4/5ka6asw CACHE FILEPATH "")
```

Comment:
would you remind me why this path was hardcoded? it seems nc-config works, but nf-config does not?

Reply:
Couldn't imagine why one worked but the other didn't but running with line 24 uncommented instead of 27 results in the behavior described in line 26, where it tries to grab a different version that doesn't work with the correct version of netcdf-c getting pulled.